### ISSUE TITLE
Reference genome not decompressing, inaccessible. 

### ISSUE THREAD

**Issue by NathanTBurgess** [WGSExtract/WGSExtract.github.io#11]
I am unable to download any reference genomes. Here are the error messages. Any suggestions

![Screenshot 2022-08-07 171619](https://user-images.githubusercontent.com/69878350/183311282-e1dc1cf1-c540-438c-b50b-0ccaab0e5b59.png)
?

I am trying to convert my nebula CRAM file into a 23 and me file. I made a program myself that does this but I want to check whether my program was accurate.

**Comment by RandyHarr** [comment]
The Reference Genome downloaded just fine. The problem is the script got 
started without the path to the wintools/bin which has the bioinformatic 
tools. That should have been setup in the shell before the call to the 
get_and_process_refgenomes.sh which calls process_reference_genomes.sh. Did 
you start the script with the installer or some other means?

**Comment by NathanTBurgess** [comment]
I've reinstalled it several times using Install_Win10 batch file. I am going to try redownloading the original zip file. 
Then I'll try installing it on a virtual Linux environment. I'll let you know what happens. 

Here is the only part in the installation process that might have been outside the norm.

![Screenshot 2022-08-08 112543](https://user-images.githubusercontent.com/69878350/183457410-e01ab0ad-5dc8-4921-ab56-4d8eaaa57938.png)

**Comment by NathanTBurgess** [comment]
![Screenshot 2022-08-08 115453](https://user-images.githubusercontent.com/69878350/183460625-60ec391d-1ca0-4588-8d0e-4ad560876fe8.png)

Good sign, ;p I guess there was just an issue with the download.

**Comment by NathanTBurgess** [comment]
I still can't select it as a reference genome.

![Screenshot 2022-08-08 115808](https://user-images.githubusercontent.com/69878350/183461222-f277e34a-485e-4557-89cf-89bb6f930076.png)

**Comment by RandyHarr** [comment]
That screen is to select a new location for your reference library. In case 
you plan to move it to another disk. The tool automatically determines what 
reference it needs and then looks for it in the reference library.

**Comment by RandyHarr** [comment]
yes. the manual has lots of detail.

Randy
(from phone, DYACs involved)

**Comment by RandyHarr** [comment]
So is this issue resolved?  I am not sure I helped at all yet but ...

**Comment by NathanTBurgess** [comment]
I think so, just one more question. If my 30x nebula CRAM file comes back as 0x what is the most likely cause?

**Comment by RandyHarr** [comment]
General questions should use the Facebook group

It could be a number of things. How big is the CRAM? How many gbases does 
the tool show it has (RAW or mapped)? How many read segments? If all the 
numbers are zero, then maybe the file was downloaded incorrectly or not 
able to be read. Should report that as an error but...  If numbers are in 
the gbases and tens to hundreds of millions of read segments, maybe they 
did a WES or you ordered their 0.4x WGS product (which would show 0 in the 
tool due to rounding). Or they delivered that instead of a 30x.

Randy
(from phone, DYACs involved)

**Comment by NathanTBurgess** [comment]
Oh sorry, I'll ask questions there from now on. It is 44.5 Gb.  Thanks for all of your help!

**Comment by RandyHarr** [comment]
If the file is 44 GB, then it is most likely a 30x WGS result.  So there must be some other error if it is not reporting the proper statistics.  Do you see anything odd in the command log window?  What do the other stats on the Stats window show?

**Comment by NathanTBurgess** [comment]
I was doing this to tell whether the program I made to convert a VCF file into a 23 and me txt file worked. Mine was much more sparse. :( I was using the v3 23 and me blank template. I basically just created dictionaries and then for each entry in the 23 and me template that is also in the dictionary made from the VCF file I would write into a file in the 23 and me format. I created keys in the dictionaries using the "chr" + chromosome number/letter +0(if it is a chromosome lower than 10) + location. I generated the VCF txt file using bcftools. No need to respond to this but if you have any immediate corrections to my thinking that pop up I am all ears.
